src/pyenphase/updaters/production.py

```diff
@@ -165,9 +169,20 @@ async def update(self, envoy_data: EnvoyData) -> None:
                 if consumption:
                     phase_consumption[PHASENAMES[phase]] = consumption
 
+                net_consumption: EnvoySystemConsumption | None = (
+                    EnvoySystemConsumption.from_production_phase(
+                        production_data, phase, consumption_segment=1
+                    )
+                )
+                if net_consumption:
```

Comment:
```suggestion
                if net_consumption := EnvoySystemConsumption.from_production_phase(
                    production_data, phase, consumption_segment=1
                ):
```

It looks like `EnvoySystemConsumption.from_production_phase` already has a type hint